pkgs/fc/ceph/fc/ceph/maintenance.py

```diff
@@ -4,9 +4,8 @@
 import subprocess
 import sys
 
-import fc.util.directory
-
 import fc.ceph.images
+import fc.util.directory
```

Comment:
This is unrelated. I guess it's caused by some pre-commit upgrade?

Reply:
It's ok, this is caused by the removal of `fc/ceph/__init__.py` (which is empty). isort now sees fc.ceph as external package and sorts it differently.